From 47caa7407f9d8e165e3201f4875979940bce5532 Mon Sep 17 00:00:00 2001
From: Alexandre Belloni <alexandre.belloni@bootlin.com>
Date: Wed, 10 Apr 2019 22:56:03 +0200
Subject: [PATCH 09/44] rtc: pcf85363: remove useless forward declaration

It is not necessary to forward declare pcf85363_driver as it is not used
before being declared.

Signed-off-by: Alexandre Belloni <alexandre.belloni@bootlin.com>
---
 drivers/rtc/rtc-pcf85363.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/rtc/rtc-pcf85363.c b/drivers/rtc/rtc-pcf85363.c
index f8cdb475077d..6c8a2ee40420 100644
--- a/drivers/rtc/rtc-pcf85363.c
+++ b/drivers/rtc/rtc-pcf85363.c
@@ -112,8 +112,6 @@
 
 #define NVRAM_SIZE	0x40
 
-static struct i2c_driver pcf85363_driver;
-
 struct pcf85363 {
 	struct rtc_device	*rtc;
 	struct regmap		*regmap;
-- 
2.17.1

